src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -62,6 +100,7 @@ public DocumentBuildContext(string buildOutputFolder, IEnumerable<FileAndType> a
             _xrefServiceUrls = xrefServiceUrls;
             GroupInfo = groupInfo;
             MaxParallelism = maxParallelism;
+            MaxHttpParallelism = maxParallelism * 2;
```

Comment:
> maxParallelism * 2 [](start = 32, length = 19)

keep the same as `Math.Max(64, parameters.MaxParallelism * 2)`?

Reply:
for unit test only.